### PR TITLE
Develop to master - fix resource formats and prepare for CKAN 2.10

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Split resource format config only on line breaks, not spaces, so entries like "SHP AND KMZ" aren't split.
- Drop CKAN 2.8 support, including revisions, Pylons, and Fanstatic
- Replace obsolete 'str' validators with proper ones
- Add support for viewing activity streams via CKAN 2.10 plugin ckanext-activity
- Add CKAN 2.10 support to header template
- Initialise Data Request tables on command line if possible
- Move Faker from main requirements to test
- Refactor scenario tests to reuse more steps and not rely on stock data

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ Appium-Python-Client<=0.52
 ckanapi==4.3
 ckantoolkit>=0.0.4
 factory-boy
+Faker
```
Comment (RossWebsterWork) [inline dev-requirements.txt:7]: Should "Faker" be capitalised here?

File: test/features/organisations.feature [hunk 2]
```diff
@@ -4,39 +4,39 @@ Feature: Organization APIs
...
+        And I view the "test-organisation" organisation API "including" users
+        Then I should see an element with xpath "//*[contains(string(), '"success": true') and contains(string(), '"name": "test_org_admin"') and contains(string(), '"name": "test_org_editor"')]"
 
         Examples: Admins
-            | Persona             |
-            | SysAdmin            |
-            | Organisation Admin  |
+            | Persona       |
+            | SysAdmin      |
+            | TestOrgAdmin  |
```
Comment (RossWebsterWork) [inline test/features/organisations.feature:13]: Could remove the extra space here

File: ckanext/data_qld/tests/conftest.py [hunk 3]
```diff
@@ -199,7 +189,7 @@ def archival_init():
 @pytest.fixture
 def mock_storage(monkeypatch, ckan_config, tmpdir):
     monkeypatch.setitem(ckan_config, u'ckan.storage_path', str(tmpdir))
-    monkeypatch.setattr(uploader, u'_storage_path', str(tmpdir))
+    monkeypatch.setattr(uploader, u'get_storage_path', lambda: str(tmpdir))
```
Comment (RossWebsterWork) [inline ckanext/data_qld/tests/conftest.py:192]: How does the lambda work here?

File: dev-requirements.txt (hunk 1 above)
Comment (ThrawnCA) [reply dev-requirements.txt:7]: Both seem to work.

File: ckanext/data_qld/tests/conftest.py (hunk 3 above)
Comment (ThrawnCA) [reply ckanext/data_qld/tests/conftest.py:192]: It's replacing a function. The lambda is a placeholder function, just returning a value.

File: .docker/scripts/process-config.sh
Comment (RossWebsterWork) [inline .docker/scripts/process-config.sh:1]: I personally don't quite understand this, I think it's some sort of search and replace, but I imagine it's all good

**Review by RossWebsterWork — APPROVED** [review]

**Pull request merged**